src/Symfony/Component/HttpKernel/DataCollector/DumpDataCollector.php

```diff
@@ -164,6 +164,16 @@ public function collect(Request $request, Response $response, \Exception $except
         }
     }
 
+    public function reset()
+    {
+        $this->stopwatch->reset();
```

Comment:
the data collector dealing with Stopwatch is not this one. So it should not be responsible for resetting (the DumpDataCollector uses the stopwatch to profile itself, it does not read from it).

This should be done in the TimeDataCollector instead

Reply:
Inside `dump()` we deal with the Stopwatch in `DumpDataCollector` too: https://github.com/xabbuh/symfony/blob/dd53319950793f6bbed96e4a8ae764c48f64ebfc/src/Symfony/Component/HttpKernel/DataCollector/DumpDataCollector.php#L68

But you are right about the `TimeDataCollector`. I have updated it too.